mlir-tensorrt/compiler/lib/Dialect/Plan/IR/PlanOps.cpp

```diff
@@ -465,6 +467,68 @@ void InlineClosedGroupOp::build(OpBuilder &b, OperationState &state,
   state.addTypes(TypeRange(outs));
 }
 
+//===----------------------------------------------------------------------===//
+// InlineClosedAllocGroupOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult InlineClosedAllocGroupOp::verify() {
+  Operation *op = getOperation();
+  // Check for res_attrs
+  if (op->hasAttr("res_attrs")) {
```

Comment:
Nit: remove braces on single-statement bodies of `if` statements 

https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements